heron/spi/src/java/com/twitter/heron/spi/common/Key.java

```diff
@@ -109,21 +109,22 @@
   INSTANCE_DISK             ("heron.resources.instance.disk", ByteAmount.fromBytes(1073741824)),
 
   //keys for checkpoint management
-  IS_STATEFUL                       ("heron.is.stateful", Boolean.FALSE),
-  IS_CLEAN_STATEFUL_CHECKPOINTS     ("heron.is.clean.stateful.checkpoints", Boolean.FALSE),
-  CKPTMGR_STATEFUL_STORAGE          ("heron.statefulstorage.type", Type.STRING),
-  STATEFUL_STORAGE_CONF             ("heron.statefulstorage.config", Type.MAP),
-  STATEFUL_STORAGE_CLASSPATH        ("heron.statefulstorage.classpath", Type.STRING),
+  IS_STATEFUL_ENABLED                      ("heron.is.stateful.enabled", Boolean.FALSE),
+  IS_CLEAN_STATEFUL_CHECKPOINTS            ("heron.is.clean.stateful.checkpoints", Boolean.FALSE),
+  STATEFUL_STORAGE_CLASSNAME               ("heron.statefulstorage.classname", Type.STRING),
+  STATEFUL_STORAGE_CONF                    ("heron.statefulstorage.config", Type.MAP),
+  STATEFUL_STORAGE_CUSTOM_CLASSPATH        ("heron.statefulstorage.custom.classpath", Type.STRING),
```

Comment:
Got it. Should we change `custom` to `additional` to be consistent with `Config.TOPOLOGY_ADDITIONAL_CLASSPATH`?